### PR TITLE
[WIP] Adding a document to highlight supported capabilities within the .NET Json stack for v1.

### PR DESCRIPTION
**Pull request by ahsonkhan**
cc @joshfree, @steveharter, @bartonjs, @terrajobst, @glennc, @rynowak, @pranavkm, @BrennanConroy, @davidfowl, @KrzysztofCwalina, @stephentoub

### CODE REVIEW

File: src/System.Text.Json/docs/FeatureList.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - Effectively, the subset of types that are supported by the [Utf8Parser](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.text.utf8parser?view=netstandard-2.1)/[Formatter](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.text.utf8formatter?view=netstandard-2.1)
+- Write/Serialize formatted/indented and minimized
+- Async APIs on the (De)Serializer
+- User-defined classes with public properties with default/parameterless constructor
+- Base64 support for byte arrays
+- Support for ignoring nulls, and customize property names (such as camel casing)
+- Allow overflow JSON to round-trip (via `JsonExtensionDataAttribute`)
+- Detect (and throw) when observing cycles in the object graph
+- Support for collections such as IList, ICollection, IDictionary, etc.
+    - For deserializing dictionaries, only `string: object` and `string: JsonElement` pairs are supported
```
Comment (BrennanConroy) [inline src/System.Text.Json/docs/FeatureList.md]: Isn't `string: poco` also supported

**Comment by ahsonkhan** [comment]
cc @buyaa-n, @layomia

**Comment by maryamariyan** [comment]
@ahsonkhan we're planning on reducing the number of inactive open PRs on corefx. 
Do you know if this PR will be updated soon or would it be better to close and reopen later?

**Comment by ahsonkhan** [comment]
I'll close it and re-open later. Thanks for the reminder :)

**Pull request closed without merging**